src/index.ts

```diff
@@ -203,6 +203,7 @@ export async function isAvailable() {
         'ENETUNREACH',
         'ENOENT',
         'ENOTFOUND',
+        'ECONNREFUSED',
```

Comment:
Is there a complete list of these somewhere?  Or are there specific ones we're trying to avoid?  Wondering if we just keep adding things to this list as people report them, or if there's a better way.

Reply:
I feel like I tried to dig into this once before without much success, can dig around again.